tests/transports-integrations/tests/integrations/test_google.py

```
@@ -32,12 +32,16 @@
     IMAGE_URL,
     BASE64_IMAGE,
     INVALID_ROLE_MESSAGES,
+    STREAMING_CHAT_MESSAGES,
```

Ruff (0.12.2)

`..utils.common.STREAMING_CHAT_MESSAGES` imported but unused

Remove unused import

(F401)